app/lib/profile/pages/profile.dart

```diff
@@ -1,12 +1,31 @@
-import 'package:flutter/material.dart';
+import 'package:flutter/material.dart' hide MetaData;
+
+import '../../common/models/metadata.dart';
+import '../../common/models/userdata.dart';
 
 class ProfilePage extends StatelessWidget {
   const ProfilePage({Key? key}) : super(key: key);
 
+// ignore: flutter_style_todos
+// TODO: remind me to remove this, for debugging only!
```

Comment:
```suggestion
// ignore: flutter_style_todos
// TODO(Toalaah): remind me to remove this, for debugging only!
```

You can actually make a valid TODO that the linter wont complain about like this